packaging/debian/changelog

```diff
@@ -0,0 +1,5 @@
+ansible-runner (1.2.0-1) unstable; urgency=low
+
+  * 1.2.0 release
```

Comment:
This will probably go out on 1.3.0 first.

Reply:
I've bumped it now to 1.3.0.